package.json

```diff
@@ -58,13 +65,13 @@
     "jsdom": "11.11.0",
     "jsdom-global": "3.0.2",
     "jsx-pragmatic": "^2.0.7",
-    "mocha": "^4.0.1",
+    "mocha": "^8.3.1",
     "mocha-sinon": "^2.1.0",
     "nodemon": "^1.19.1",
     "nyc": "^14.1.1",
     "sinon": "^7.3.2",
     "smoothscroll-polyfill": "^0.4.4",
-    "uglifyjs-webpack-plugin": "^1.3.0",
+    "uglifyjs-webpack-plugin": "^1.1.2",
```

Comment:
This is intentional I presume

Reply:
npm audit fix did that, for some reason. I remove the dependency as it is not needed anymore.